astropy/coordinates/angles/formats.py

```diff
@@ -399,42 +399,28 @@ def parse_angle(angle, unit=None, debug=False):
     return _AngleParser().parse(angle, unit, debug=debug)
 
 
-def degrees_to_dms(d):
+def decimal_to_sexagesimal(a, /):
```

Comment:
```suggestion
def _decimal_to_sexagesimal(a, /):
```

Reply:
done ! (I did it manually so I could change the definitions and call sites in one go)